src/lib/account-profile/account-profile.component.html

```diff
@@ -0,0 +1,10 @@
+<button mat-icon-button [matMenuTriggerFor]="accountMenu">
+  <mat-icon>person</mat-icon>
+</button>
+
+<mat-menu #accountMenu="matMenu" [overlapTrigger]="false">
```

Comment:
What's `matMenu` used for?

Reply:
this is containing component for the `account-profile-menu` and `button` that triggeers the mat menu. like the app switcher does